ts/output/common/Wrapper.ts

```diff
@@ -314,14 +314,34 @@ export class CommonWrapper<
     return bbox;
   }
 
+  /**
+   * Return the wrapped node's bounding box that includes borders and padding
+   *
+   * @param {boolean} save  Whether to cache the bbox or not (used for stretchy elements)
+   * @return {BBox}  The computed bounding box
+   */
+  public getOuterBBox(save: boolean = true): BBox {
+    const bbox = this.getBBox(save);
+    if (!this.styles) return bbox;
+    const obox = new BBox();
+    Object.assign(obox, bbox);
+    for (const [name, side] of BBox.StyleAdjust) {
+      const x = this.styles.get(name);
+      if (x) {
```

Comment:
`x` is never used. Collapse to
```suggestion
      if (this.styles.get(name)) {
```

Reply:
OOPS!, meant to replace the duplicate call in the next line.